metadata.yaml

```diff
@@ -20,7 +20,8 @@ provides:
 storage:
   mongodb:
     type: filesystem
-    location: /data/db
+    # mongod is strictly confined in the charmed-mongodb snap
+    location: /var/snap/charmed-mongodb/common/var/lib/mongod
```

Comment:
Would it make more sense for this to be `/var/snap/charmed-mongodb/common/var/lib/mongodb` instead to mirror the deb package?  It would essentially be `$SNAP_COMMON/var/lib/mongodb` where the deb is `/var/lib/mongodb`

Reply:
lets use `/var/lib/mongodb`, but for data dir only.
config dir should remain the same